site/src/templates/languagePage.tsx

```diff
@@ -0,0 +1,99 @@
+import { graphql } from "gatsby"
+import { MDXRenderer } from "gatsby-plugin-mdx"
+import * as React from "react"
+import ContentMarkdownSection from "../components/ContentMarkdownSection"
+import Footer from "../components/Footer"
+import Layout from "../components/Layout"
+import PageHeader from "../components/PageHeader"
+import {
+  Content,
+  Sidebar,
+  SidebarAndContent,
+  SidebarTab,
+} from "../components/SidebarAndContent"
+import { sortSidebarTabs, stripTrailingSlash } from "./util"
+
+const renderSidebarTabs = (pages, currentPath, sectionName) => {
+  return pages.map((node, i) => (
+    <SidebarTab
+      href={node!.node!.fields!.slug}
+      active={
+        stripTrailingSlash(node!.node!.fields!.slug) ===
+        stripTrailingSlash(currentPath)
+      }
+      key={`sidebarTab-${sectionName}-${i}`}
+    >
+      {node!.node!.frontmatter!.navTitle}
+    </SidebarTab>
+  ))
+}
+
+export default ({ data, pageContext, location }) => {
+  const md = data.mdx
+  const allPages = data.allMdx.edges
+  const overviewPage = allPages.filter(
+    el => el.node.frontmatter.navTitle === "Overview"
+  )
+  const pagesWithoutOverview = sortSidebarTabs(
+    allPages.filter(el => el.node.frontmatter.navTitle !== "Overview")
+  )
+
+  const currentPath = location.pathname
+
+  const LanguagePageHeader = (
+    <PageHeader
+      headingText={md.frontmatter.title}
+      summaryParagraph={md.frontmatter.summaryParagraph}
+      tags={md.frontmatter.tags}
+    />
+  )
+
+  return (
+    <Layout
+      pageTitle={md.frontmatter.title}
+      currentPath={currentPath}
+      pageHeader={LanguagePageHeader}
+      footer={<Footer />}
+    >
+      <SidebarAndContent>
+        <Sidebar>
+          {renderSidebarTabs(overviewPage, currentPath, "language")}
+          {renderSidebarTabs(pagesWithoutOverview, currentPath, "language")}
+        </Sidebar>
+        <Content>
+          <ContentMarkdownSection>
+            {/*
+            // @ts-ignore */}
```

Comment:
Not a huge thing, but did this get borked by prettier?

Reply:
Looks like it, yep. Bah